### PR TITLE
PD-108 Update Alerts and SMTP Content

### PR DESCRIPTION
**Pull request by micjohnson777**
This commit eliminates the tabs in the AlerManage.md article, adds the TOC, updates the information in the SMTP and email configuration section. Adds the extra front matter parameters, adds tags and taglists, and the children shortcode to the index file. This does not break the tutorials out into separate articles. That is deferred to the TC content reorg effort for later this year or when TC 3.0 is released.

Thanks for contributing to TrueNAS documentation! By opening a Pull Request, you're acknowledging that your changes will be distributed under the [Creative Commons 4.0](https://creativecommons.org/licenses/by-nc-sa/4.0/) license.

### CODE REVIEW

File: content/TrueCommand/Alerts/AlertManage.md [hunk 1]
```diff
@@ -33,7 +42,7 @@ Alerts generated by a monitored system display on both the **Systems** screen an
 
 ![DashboardSystemAlerts](/images/TrueCommand/2.0/DashboardSystemAlerts.png "Dashboard System Alerts")
 
-To view all alerts for a single system from the **Dashboard**, click on the system name to open the system details screen. The **Alerts** area displays the system's alerts.
+To view all alerts for a single system from the **Dashboard**, click on the system name to open the system details screen. The **Alerts** area displays the systems alerts.
```
Comment (clairewrtr) [inline content/TrueCommand/Alerts/AlertManage.md:45]:
```suggestion
To view all alerts for a single system from the **Dashboard**, click on the system name to open the system details screen. The **Alerts** area displays the system's alerts.
```

File: content/TrueCommand/Alerts/AlertManage.md [hunk 2]
```diff
@@ -88,24 +96,22 @@ To create a new alert rule, click **+ NEW ALERT RULE** and follow the creation w
...
+2. Specify the **Alert Triggers**.
+   Select a data source or rule type from the dropdown list to determine what can trigger an alert. For example, *cpu_temperature* means the alert rule monitors the temperature of the chosen system. Scroll down the list to find the desired source.
 
    ![AlertTriggerDataSources](/images/TrueCommand/2.1/AlertTriggerDataSources.png "Alert Trigger Data Sources")
 
-   b. Select the comparison type from the **Comparator** drop-down list (**Greater Than**, **Less Than**, or **Not Equals**). The comparison type applies to the data source and comparison value.
+   b. Select the comparison type from the **Comparator** dropdown list (**Greater Than**, **Less Than**, or **Not Equals**). The comparison type applies to the data source and comparison value.
    c. Specify the comparison value by entering an integer appropriate for the selected options in the **Value** field. The integer acts as a threshold or limitation for when the rule generates an alert.
```
Comment (clairewrtr) [inline content/TrueCommand/Alerts/AlertManage.md:111]: Line 111 item "c." not breaking to new line.

File: content/TrueCommand/Alerts/Colors.md [hunk 3]
```diff
@@ -1,6 +1,11 @@
 ---
-title: "Colors"
+title: "Themes and Colors"
+description: "This article provides bacic information on customizing themes or the alert colors to the suit user preference."
```
Comment (clairewrtr) [inline content/TrueCommand/Alerts/Colors.md:3]:
```suggestion
description: "This article provides basic information on customizing themes or the alert colors to the suit user preference."
```

File: content/TrueCommand/Alerts/_index.md [hunk 4]
```diff
@@ -8,3 +8,5 @@ TrueCommand allows for user notification based on custom defined alerts for conn
 Method of alert notifications as well as theming of alerts in the TrueCommand interface can be user customized.
 
 {{< include file="static/includes/General/MenuNav.md.part" markdown="true" >}}
+
+{{< children description="true" >}}
```
Comment (clairewrtr) [inline content/TrueCommand/Alerts/_index.md:12]:
```suggestion
{{< children depth="2" description="true" >}}
```

**Review by clairewrtr — DISMISSED** [review]
Looks good, minor changes.

File: content/TrueCommand/Alerts/AlertManage.md (hunk 1 above)
Comment (micjohnson777) [reply content/TrueCommand/Alerts/AlertManage.md:45]: we do not use possessive case.

File: content/TrueCommand/Alerts/AlertManage.md (hunk 2 above)
Comment (micjohnson777) [reply content/TrueCommand/Alerts/AlertManage.md]: Fixed "a" that was not showing and "c" in a new commit.

**Review by Mrt134 — APPROVED** [review]

**Pull request merged**